atlas/packages/app/src/views/ChannelView/ChannelView.style.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled'
+import { Avatar } from '@/shared/components'
+import theme from '@/shared/theme'
+
+type ChannelHeaderProps = {
+  coverPhotoURL: string
+}
+export const Header = styled.section<ChannelHeaderProps>`
+  background-image: linear-gradient(0deg, #000000 10.85%, rgba(0, 0, 0, 0) 88.35%),
+    ${(props) => `url(${props.coverPhotoURL})`};
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+  height: 430px;
+  padding: 0 ${theme.sizes.b8}px;
+`
+export const TitleSection = styled.div`
+  display: flex;
+  align-items: center;
+  padding-top: ${theme.sizes.b10}px;
+`
+export const Title = styled.h1`
+  font-size: ${theme.typography.sizes.h2}px;
```

Comment:
`typography.sizes.h2` already specifies a unit so adding the `px` at the end actually breaks the style. This may be my fault because I introduced `sizes` which are number values and now we get these mixed up. We probably should clean that up soon